src/FSharpFakeTargets/targets.fs

```diff
@@ -157,14 +171,39 @@ module Targets =
           _IncrementAssemblyInfo datNET.Version.IncrMajor
       )
 
+    let prereleaseTargetHelper preStr =
+      _IncrementAssemblyInfo (fun verStr ->
+        verStr
+        |> datNET.SemVer.parse
+        |> datNET.SemVer.mapPre (fun _ -> preStr)
+        |> datNET.SemVer.stringify
+      )
+
+    let setPreReleaseTarget parameters =
+      _CreateTarget "SetPrerelease:RootAssemblyInfo" parameters (fun _ ->
+        let preStr =
+          match getBuildParam "pre" with
+          | "" -> None
```

Comment:
Unfortunately we can't, because this is in a match expression.
The compiler gives `This field is not a literal and cannot be used in a pattern` when I try `String.Empty` here.

Sorry.